File: 2806-account-balance-after-rounded-purchase/2806-account-balance-after-rounded-purchase.py
class Solution:
    def accountBalanceAfterPurchase(self, purchaseAmount: int) -> int:
        rem = purchaseAmount % 10
        if rem < 5:
            purchaseAmount = purchaseAmount - rem
        else:
            purchaseAmount = purchaseAmount + (10 - rem)

        return 100 - purchaseAmount

File: 2806-account-balance-after-rounded-purchase/2806-account-balance-after-rounded-purchase.js
var accountBalanceAfterPurchase = function(purchaseAmount) {
    let rem = purchaseAmount % 10;
    if (rem < 5)
        purchaseAmount = purchaseAmount - rem;
    else
        purchaseAmount = purchaseAmount + (10 - rem);

    return 100 - purchaseAmount;
};